CMakeLists.txt

```diff
@@ -112,8 +115,7 @@ if (BUILD_SUITE)
 	else (PLX_FOUND OR PXI_FOUND)
 		#Find the PLX Library
 	   find_package(PLX REQUIRED)
-   	link_directories(${PLX_LIBRARY_DIR})
-	   #add_definitions("-DPLX_LINUX -DPCI_CODE -DPLX_LITTLE_ENDIAN")
```

Comment:
If we've gotten this far without it, then I suspect that we do not need it. I would prefer not to have blocks of code commented out.